### ISSUE TITLE
Update dependencies to latest versions

### ISSUE THREAD

**Issue by roelnoort** [kozzztya/node-helpers#1]
Hi,

I am using this library. Indirectly via feathers-amqp-events.
It is giving me some npm issues with vulnerabilties.
These can be solved by updating to latest versions of dependencies.
The tests still pass after update.

If you agree, could you consider updating this please?

Suggested new versions for dependencies in `package.json`:
```
  "dependencies": {
    "amqplib": "^0.7.1",
    "axios": "^0.21.1",
    "babel-runtime": "6.26.0",
    "bluebird": "^3.7.2",
    "bluebird-retry": "^0.11.0",
    "debug": "^4.3.1",
    "deep-extend": "^0.6.0",
    "install": "^0.13.0",
    "lodash": "^4.17.21",
    "object-recursive-iterator": "^2.1.0",
    "require-dir": "^1.2.0"
  }
```

Thanks, Roel

**Comment by kostia-official** [comment]
Hello, Roel, if you create a PR I can merge and publish these updates.

**Comment by roelnoort** [comment]
Thanks! Just created the PR.

**Comment by kostia-official** [comment]
Published to node-helpers@1.2.0, sorry for the late answer
